src/style-sheets/modules/_colors.scss

```diff
@@ -10,6 +10,7 @@ $olLTAccent: #43a850;
 $olLTHeader: #262626;
 $olNavLink: #232936;
 $olLTCopy: #2b2d31;
+$olLightCopy: #777777;
```

Comment:
so the variable before this is olLightThemeCopy. Is that different than olLightCopy?

Reply:
There really isn't I was just struggling to come up with a variable name! Suggestions?

Reply:
What about changing the name of both variables. Maybe olLTSolidCopy and olLTLightCopy or olLTPrimaryCopy and olLTSecondaryCopy?